extra-perl/perl-parse-yapp-driver/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME="perl-parse-yapp-driver"
+PKGDES="Perl/CPAN Module Parse::Yapp : Generates OO LALR parser modules"
```

Comment:
Perl extension for generating and using LALR parsers

(Again, stop, copying, from, Arch Linux! Their package descriptions are inconsistent at best, and why not just reference the upstream?)